### DISCUSSION TITLE
How to convert the date to French format in email reception ?

### DISCUSSION THREAD

**Discussion by VR-Webmaster** [PHPMailer/PHPMailer#3007] [Q&A]
Hello,

I added a date request in my form :

`<input type="date" id="date" name="date" value="" min="2023-01-01" max="2024-12-31" />`

But I receive the email with the date in English format (yyyy/mm/dd) 
How can I receive the date to the French format (dd/mm/yyyy) in email  reception ?
thx

**Comment by Synchro** [comment]
PHPMailer doesn't have anything to do with this - it just sends whatever you put in the body, so it's up to you. PHP has [functions to help process dates](https://www.php.net/manual/en/datetimeimmutable.createfromformat.php) though, so you should be able to do something like this:

```php
$date = DateTimeImmutable::createFromFormat('Y/m/d', $_REQUEST['date']);
$frenchDate = $date->format('d/m/Y');
```
Then insert that into whatever you pass into the PHPMailer `Body` property.

**Reply by VR-Webmaster** [reply]
Thank you for your answer !
I will try it
